Automattic/jetpack
# Search: Fix small product defects with Customberg and the dashboard

#### Changes proposed in this Pull Request:
* Correctly aligns the link buttons in the Customberg sidebar:
> <img width="251" alt="Screen Shot 2022-03-01 at 4 36 53 PM" src="https://user-images.githubusercontent.com/4044428/156267015-f337e264-ecd1-483c-a907-5a1dcecc6a12.png">

* Changes the "Edit sidebar widgets" link to `/wp-admin/widgets.php`, which increases compatibility for sites that may have the Customizer disabled by running block-based themes.





#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Apply this change to your site with a Search subscription.
* Navigate to the search dashboard and ensure that the "Edit sidebar widgets" button directs you to `/wp-admin/admin.php?page=jetpack-search`.
* Navigate to Customberg and click on the "Jetpack Search" tab on the right sidebar. Ensure that the link buttons are correctly aligned.
